src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -30,13 +32,27 @@ const MapDiv = styled.div`
   height: 100%;
 `;
 
+// Three years before the current date
+const breakpointDate = new Date().setFullYear(new Date().getFullYear() - 3);
+
+export enum MapViews {
```

Comment:
⛏️   I know it's not explicitly related to redux but we might want to put this in a types file